helm/azure-operator-resource-chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+clusterName: "test-cluster"
+commonDomain: ""
+commonDomainResourceGroup: ""
```

Comment:
WDYT about putting here something like

```
commonDomain: "test-host-cluster.westeurope.azure.gigantic.io"
commonDomainResourceGroup: "test-host-cluster"
```

Then it's easy to just replace the value with the actual cluster name. The same thing you didi for `sshPublicKey`